mmhlpb.c

```diff
@@ -710,6 +718,16 @@ H("see the 21-Dec-2017 entry in http://us.metamath.org/mpeuni/mmnotes.txt .");
 H("");
 
 
+g_printHelp = !strcmp(saveHelpCmd, "HELP VERIFY STS");
+H("Syntax:  VERIFY STS <format>");
+H("");
+H("This command error-checks that the STS rules definition covers all syntax");
+H("defined in the Metamath source file loaded. It runs through all non");

```

Comment:
Double space here !  And I would write "non-definitional" with a hyphen, or even "nondefinitional", since "non" is not a word.